src/datasets/utils/resources/tasks.json

```diff
@@ -77,7 +77,8 @@
         "subtasks": [
             "extractive-qa",
             "open-domain-qa",
-            "closed-domain-qa"
+            "closed-domain-qa",
+            "visual-question-answering",
```

Comment:
```suggestion
            "visual-question-answering"
```

Reply:
To add this task on the Hub, please open a PR to update https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts first